### DISCUSSION TITLE
Cors issue with Nodit

### DISCUSSION THREAD

**Discussion by aeither** [aptos-labs/aptos-developer-discussions#463] [Questions]
### Describe your question in detail.

when I use nodit I get cors issue with ngrok. on nodit dashboard i have it to allow all domains.

const aptos = new Aptos(
  new AptosConfig({
    network: NETWORK,
    fullnode: `https://aptos-testnet.nodit.io/${import.meta.env.VITE_NODIT_API_KEY}/v1`,
    indexer: `https://aptos-testnet.nodit.io/${import.meta.env.VITE_NODIT_API_KEY}/v1/graphql`,
    faucet: "https://faucet.testnet.aptoslabs.com",
  })
)

![screenshot](https://github.com/user-attachments/assets/f712698d-7ad5-49c8-97d6-fea902bc943e)

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

REST API

**Comment by 0x-j** [comment]
Can you try contacting nodekit? Or you can use Aptos build
https://developers.aptoslabs.com/manage/projects
